spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, :type => :controller do
+
+  let(:valid_attributes) { {title: 'Lorem', body: 'Ipsum'} }
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.